connect-inject/envoy_sidecar_test.go

```diff
@@ -203,3 +204,187 @@ func TestHandlerEnvoySidecar_NamespacesAndAuthMethod(t *testing.T) {
 && /consul/connect-inject/consul logout \
   -token-file="/consul/connect-inject/acl-token"`)
 }
+
+func TestHandlerEnvoySidecar_Resources(t *testing.T) {
+	mem1, err := resource.ParseQuantity("100Mi")
```

Comment:
This could be `resource.MustParse` that you used in the init container logic. That way we don't need to check for errors.

Reply:
🙏